airbyte-integrations/connectors/source-jdbc/src/main/java/io/airbyte/integrations/source/jdbc/AbstractJdbcSource.java

```diff
@@ -437,13 +428,11 @@ private static void assertColumnsWithSameNameAreSame(String schemaName, String t
   }
 
   @SuppressWarnings("OptionalUsedAsFieldOrParameterType")
-  private List<TableInfoInternal> discoverInternal(final JdbcDatabase database,
-                                                   final Optional<String> databaseOptional,
-                                                   final Optional<String> schemaOptional)
+  private List<TableInfoInternal> discoverInternal(final JdbcDatabase database, final Optional<String> databaseOptional)
```

Comment:
I'm wondering if this `databaseOptional` is not unused as the schema parameter too? (depending on the source database, we sometimes use it as a schema replacement)

Reply:
I think you may be right. Will take a second look after!